packages/autorest.typescript/test/integration/generated/appconfiguration/package.json

```diff
@@ -70,7 +68,8 @@
     "unit-test:browser": "echo skipped",
     "integration-test": "echo skipped",
     "integration-test:node": "echo skipped",
-    "integration-test:browser": "echo skipped"
+    "integration-test:browser": "echo skipped",
+    "update-snippets": "echo skipped"
```

Comment:
May I know what is the purpose of this script?

Reply:
This is to extract snippets used in the README.md from a test file so that we don't have to copypaste them manually to check

Reply:
Here's the dev-tool command that some packages in our JS repo already use: https://github.com/Azure/azure-sdk-for-js/blob/0cb51b47f1aa5c0b6273bfdeb3cff4cc41460ff2/common/tools/dev-tool/src/commands/run/update-snippets.ts#L17

Reply:
@jeremymeng I notice `update-snippets` is always skipped, do we have any document on when and how to use this script?

Reply:
@MaryGao ideally we would have document the command in dev-tool's readme.  Not sure about current status.  The PR to add this command also has information: https://github.com/Azure/azure-sdk-for-js/pull/31143